aws/resource_aws_backup_vault.go

```diff
@@ -18,6 +18,9 @@ func resourceAwsBackupVault() *schema.Resource {
 		Read:   resourceAwsBackupVaultRead,
 		Update: resourceAwsBackupVaultUpdate,
 		Delete: resourceAwsBackupVaultDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
To properly acceptance test this new import functionality, we should ensure there is `ImportStateVerify` testing added to all the resource acceptance tests in `aws/resource_aws_backup_vault_test.go`, e.g.

```go
{
	ResourceName:      "aws_backup_vault.test",
	ImportState:       true,
	ImportStateVerify: true,
},
```

See also this line item in the [contributing guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#resource-contribution-guidelines). 👍 

> Implements Import Acceptance Testing and Documentation: Support for resource import (Importer in resource schema) must include ImportState acceptance testing (see also the Acceptance Testing Guidelines below) and ## Import section in resource documentation.

Reply:
Oh, I overlooked it 😅 Thanks for detailed guidance, I've added tests.